File: cpp/UVA272.cpp
//https://zerojudge.tw/ShowProblem?problemid=c007
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    char c;
    bool open = true;

    while (cin.get(c)) {
        if (c == '"') {
            cout << (open ? "``" : "''");
            open = !open;
        } else {
            cout << c;
        }
    }

    return 0;
}

File: java/UVA272.java
//https://zerojudge.tw/ShowProblem?problemid=c007
import java.util.*;

public class UVA272 {
	public static void main(String args[]) {
		Scanner sc = new Scanner(System.in);
		boolean first = true;

		while(sc.hasNext()) {
			String s = sc.nextLine();
			for (int index = 0; index < s.length(); index++){
				if(s.charAt(index)=='"'){
					if(first){
						System.out.print("``");
						first = false;
					}
					else{
						System.out.print("''");
						first = true;
					}
				}
				else{
					System.out.print(s.charAt(index));
				}
			}
			System.out.println();
		}
	}
}
